packages/governance/src/committee.js

```diff
@@ -179,6 +179,9 @@ export const start = (zcf, privateArgs, baggage) => {
       getPublicFacet() {
         return publicFacet;
       },
+      shutdown() {
+        zcf.shutdown('EC Charter Shutdown');
+      },
```

Comment:
Let's not add this now. The Committee is the sole holder of the set of questions that have been voted on, and I'd prefer we find a way to write this to vstorage before killing the vats.

The reason to do something like this is to prevent ex-members from voting, so let's do that directly. Try adding a method like `stopAddingVotes()` that sets a flag that blocks use of `addQuestion()` or `getPoserInvitation()`.

Reply:
noted as a design choice in

 - #10137

Reply:
If we're going to add this method, it should be tested in unit tests and some test that does an upgrade. Either a bootstrap test or a3p-integration would do.